templates/data-providers/utils/generate-form-message.jsx

```diff
@@ -35,7 +36,11 @@ const generateFormMessage = ({ created, duplicated, error }) => {
           >
             {created.email}
           </Link>
-          .
+          . Join the community and add a{' '}
+          <Link href={HARVESTED_BY_CORE} title="Harvested by CORE Logo">
+            harvested by CORE
+          </Link>
+          badge on your website.
```

Comment:
```suggestion
           badge on your website.
```
Missing a space

Reply:
Nope, it wrong according to `eslint`. Sam fix works

Reply:
ok, maybe needs a `&nbsp;`?

Even if the linter disagree it is missing a space :)

Reply:
@mcancellieri Thanks for spotting. You might be looking at a old commit though.
It was fixed in [35a824c](https://github.com/oacore/search/pull/187/commits/35a824c9e8bbcbf4e25c94b92e68320bd2c15013) which was included in @valerapro merge